server/handler.go

```diff
@@ -236,40 +236,24 @@ func DeleteAllInstanceRequests(c *gin.Context) {
 }
 
 func GetAWSData(c *gin.Context) {
-	// read config file
-	configFile, err := os.ReadFile("config.json")
-	if err != nil {
-		log.Printf("Failed to read config file: %v", err)
-		abortWithLog(c, http.StatusInternalServerError, err)
-		return
-	}
+	// read env variable
+	configEnv := os.Getenv("MY_AMI_ATTR")
+	regionEnv := os.Getenv("MY_REGION")
 
-	// parse config file
-	var config models.Config
-	if err := json.Unmarshal(configFile, &config); err != nil {
-		log.Printf("Error parsing config file: %v", err)
-		abortWithLog(c, http.StatusInternalServerError, err)
+	config := models.Config{}
+
+	err := json.Unmarshal([]byte(configEnv), &config)
+	if err != nil {
+		log.Printf("Failed to describe unmarshal ec2 configuration: %v", err)
```

Comment:
You can also set it to the Error level by using the `Errorf` function.

```suggestion
		log.Errorf("Failed to describe unmarshal ec2 configuration: %v", err)
```

Reply:
Alright, will make that change

Reply:
might take some time since it's a different package, I'll do this in the future and prioritize other things first.